templates/service-template.yml

```diff
@@ -967,6 +967,7 @@ objects:
             - --aws-secret-access-key-file=/secrets/service/aws.secretaccesskey
             - --aws-route53-access-key-file=/secrets/service/aws.route53accesskey
             - --aws-route53-secret-access-key-file=/secrets/service/aws.route53secretaccesskey
+            - --gcp-api-credentials-file=/secrets/service/gcp.api-credentials
```

Comment:
when adding the credentials to vault for stage/prod. The key has to be `gcp.api-credentials`